dsc/db/models.py

```diff
@@ -144,3 +147,35 @@ def create(
                 ) from exception
             raise
         return item
+
+    @classmethod
+    def get_or_create(
+        cls, item_identifier: str, batch_id: str, workflow_name: str
+    ) -> "ItemSubmissionDB | None":
+        log_details = (
```

Comment:
A brief docstring outlining the logic could be helpful here

Reply:
Commits got a bit messy, but a [docstring has been added](https://github.com/MITLibraries/dspace-submission-composer/blob/IN-1316-update-reconcile-command-to-use-dynamodb/dsc/db/models.py#L176-L183).